k8s/helm/values.yaml

```
@@ -0,0 +1,9 @@
+manager:
+  image:
+    repository: cloudogu/k8s-ces-control
+    tag: 0.4.0
+  imagePullPolicy: IfNotPresent
+  replicas: 1
+  env:
+    stage: production
+    logLevel: info
```

```suggestion
    logLevel: info

```
add newline, so that the file does not change, when processed by the make-targets